walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +270,103 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
+
+  @Test
+  public void noEventsToOccupy() {
+    // Pass in no events. This should result in no slots being considered as occupied.
+
+    Collection<Event> events = NO_EVENTS;
+    Event[] eventsArray = events.toArray(new Event[events.size()]);
+
+    MeetingRequest request = new MeetingRequest(Arrays.asList(PERSON_B), DURATION_30_MINUTES); 
+
+    boolean[] actual = query.getOccupiedSlotsInDay(eventsArray, request);
+    boolean[] expected = new boolean[48];
```

Comment:
Relying on default initialization can be tricky, unless this is a stable interface that will never change. It should work for the purpose here.